metrics/storage/fio_job.sh

```diff
@@ -29,15 +29,16 @@ TEST_ARGS="$@"
 # These values are a basic configuration for a FIO job
 # execution, however they could be overridden in order to
 # run a customized FIO job.
-BLOCK_SIZE="4k"
+BLOCK_SIZE="16k"
 ENGINE="sync"
 FILE_SIZE="4G"
 FIO_TEST_NAME="io_test"
 NUM_JOBS="1"
 OPERATION="randread"
 TEST_NAME="storage fio test"
 EXTRA_ARGS=""
-TEST_TIME="30"
+TEST_TIME="300"
+IO_DEPTH="1024"
```

Comment:
While it is nice to have this option, `iodepth` will be always 1 in case of `sync` engine. `iodepth` (number of concurrent requests) only makes a difference in case of `libaio` engine.

Reply:
Running test for a longer time does produce reproduceable results. But they do take some time now (around 7*4 min).
We may have to live with this initially, but we really should consider preparing an image with requirements installed and using that instead going forward to cut down on the test time.

@GabyCT Can you try doing a few runs with with FILE_SIZE=4G and taking out TEST_TIME and --timebased all together. This may perform faster on the xeon that we use for testing while still making sure we transfer 4G of data which is greater than the 2G of memory that we start the VM with.